docs/wiki/99_Developers/LORIS-scripts-in-the-tools--directory.md

```diff
@@ -1,44 +1,20 @@
-**[[HOME|Home]]** > **[[SETUP|Setup]]** > **[[LORIS SCRIPTS IN THE TOOLS/ DIRECTORY|LORIS scripts in the tools  directory]]**
-
-***
-
-- [assign_missing_instruments.php](#assign_missing_instrumentsphp)
-- [config_to_db.php [DEPRECATED]](#config_to_dbphp-deprecated)
-- [delete_candidate.php](#delete_candidatephp)
-- [detect_conflicts.php](#detect_conflictsphp)
-- [fix_timepoint_date_problems.php](#fix_timepoint_date_problemsphp)
-- [generate_tables_sql.php](#generate_tables_sqlphp)
-- [populate_visit_windows.php](#populate_visit_windowsphp)
-- [quickform_parser.php](#quickform_parserphp)
-- [score_instrument.php](#score_instrumentphp)
-
-***
-
-This section is under construction.
-
+# Scripts in Tools/ directory
 > Note: scripts may generate PHP Notice messages, which is normal and should not affect output.
 
 **See Also**:  CouchDB* loading scripts -> [Loading the Data Querying Tool](https://github.com/aces/Loris/wiki/Data-Querying-Tool#loading-the-data-query-tool)
```

Comment:
this link needs to be updated. @haoweiqiu could you also triple-check other links throughout this PR to ensure they're up to date?  thank you

Reply:
I noticed this issue while while transferring the docs. However, the updated Data Querying Tool page is provided in PR #7079 . So, I was thinking that maybe I could change this link after PR #7079 is merged? Meanwhile, other links in this PR are updated.